### DISCUSSION TITLE
The Pragmatic Programmer [LIDO] ✅

### DISCUSSION THREAD

**Discussion by gustavoferrazfontes** [pilhacheia/clube-do-livro-dev#7] [Books]
Está na minha minha wishlist a algum tempo. Seria interessante fazer essa leitura em um grupo.
[Livro na amazon](https://www.amazon.com.br/Pragmatic-Programmer-journey-mastery-Anniversary-ebook/dp/B07VRS84D1/)

**Comment by lincolixavier** [comment]
Boa! Eu não terminei ainda. Mas seria ótimo começar com esse, é basicamente senso comum no nosso mundo. Dave thoma é incrível.

**Comment by luiznasciment0** [comment]
Esse parece ser fantástico, inclusive a minha cópia está a caminho!

**Comment by gustavoferrazfontes** [comment]
Alguem tem alguma sugestão de como organizar os encontros? Será quinzenal, mensal ou na conclusão do livro?

**Reply by lincolixavier** [reply]
Acho quinzenal uma boa, e um encontro no final.

**Comment by lincolixavier** [comment]
Que livro absurdamente incrível!!
